cmd/collection/main.go

```diff
@@ -117,6 +126,12 @@ func main() {
 				"additional fraction of replica timeout that the primary will wait for votes")
 			flags.DurationVar(&blockRateDelay, "block-rate-delay", 250*time.Millisecond,
 				"the delay to broadcast block proposal in order to control block production rate")
+
+			// epoch qc contract flags
+			flags.StringVar(&accessAddress, "access-address", "", "the address of an access node")
+			flags.StringVar(&qcContractAddress, "qc-contract-address", "", "the address of the Epoch QC contract")
+			flags.StringVar(&privateKey, "private-key", "", "the private key of the account used to interact")
```

Comment:
what account is this private key for? 

Usually, the private key is managed by a separate module, instead of passing as a clear text.

Reply:
Good point, it might a good idea to include this key in the private node info file we load from disk:
https://github.com/onflow/flow-go/blob/cf6652aa76baca375a2685c6fa90f358d48f799d/cmd/scaffold.go#L805-L814